### ISSUE TITLE
this dataset with out intensity

### ISSUE THREAD

**Issue by PeterJaq** [LiDAR-Perception/LiDAR-CS#7]
hello, when I use this dataset, I found all intensity in bin is 1.0.

**Comment by fangjin-cool** [comment]
> hello, when I use this dataset, I found all intensity in bin is 1.0.

Yes, our dataset was generated through simulation, and we found that the intensity readings from different models of LiDAR devices are inconsistent. Therefore, for uniformity within the dataset, we have set the intensity value to 1.0 across all settings. Additionally, we encourage the researchers to focus on the distribution of the point cloud rather than the intensity values.
